dist/livenessprobe.js

```diff
@@ -0,0 +1,34 @@
+const pm2 = require('pm2');
+let ports = require('./ecosystem-ports').ports;
+
+if (process.env.ACTIVE_THEMES) {
+  const active = process.env.ACTIVE_THEMES.split(',');
+  ports = Object.entries(ports)
+    .filter(([theme]) => active.includes(theme))
+    .reduce((acc, [k, v]) => ({ ...acc, [k]: v }), {});
+}
+
+pm2.connect(err1 => {
+  if (!err1) {
+    pm2.list((err2, list) => {
+      if (!err2) {
+        Object.entries(ports).forEach(([theme]) => {
+          if (
+            list
+              .filter(el => el.name === theme)
+              .find(el => el.pm2_env.status === 'online' || el.pm2_env.status === 'launching')
+          ) {
+            process.exit(0);
+          }
+        });
+        process.exit(1);
```

Comment:
This will exit with 0 if the first process is found and not check other themes.

How about:
```
const allUp = Object.entries(ports).map(([theme]) => list...filter...find).reduce((acc, val) => acc && !!val, true);
process.exit(allUp ? 0 : 1);
```